core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -59,8 +63,13 @@
 
   private static final boolean JNDI_AVAILABLE = jndiAvailable();
 
+  // From https://github.com/grpc/proposal/blob/master/A2-service-configs-in-dns.md
+  private static final String SERVICE_CONFIG_NAME_PREFIX = "_grpc_config.";
+  // From https://github.com/grpc/proposal/blob/master/A5-grpclb-in-dns.md
+  private static final String GRPCLB_NAME_PREFIX = "_grpclb._tcp.";
+
   @VisibleForTesting
-  static boolean enableJndi = false;
+  static boolean enableJndi = true;
```

Comment:
We should keep this off until it is stable.

Reply:
Put it behind a flag so that it can be tested before release.